src/identity/masterpubkey.ts

```diff
@@ -184,8 +186,11 @@ export class MasterPublicKey extends Identity implements IdentityInterface {
       address: {
         confidentialAddress: confidentialAddress!,
         blindingPrivateKey: blindingKeyPair.privateKey!.toString('hex'),
+        derivationPath: `${this.baseDerivationPath}/${
```

Comment:
I understand this will eventually go away, could you make a variable that wraps both?

Reply:
yes ok